.github/workflows/build_and_test.yaml

```diff
@@ -7,6 +7,9 @@ jobs:
     name: build py3.${{ matrix.python-version }} on ${{ matrix.platform || matrix.os.base }}-${{ matrix.os.version }}
     strategy:
       fail-fast: true
+      # NOTE: In the matrix below, we call out macos 13 and 14 explicitly. This is to enable running on both Intel and Apple Silicon for Mac.
+      # As of this writing, MacOS 13 is Intel and MacOS 14 is Apple Silicon. Ideally, the Apple transition to their new chip occurs fast and
+      # we can drop macos-13 checks and then just use 'latest'.
```

Comment:
@michael-okeefe , could we use `13` and `latest` so that we don't have to remember to increment the `14`?

Reply:
@calbaker Great idea. I made the change and the tests are running.